src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -157,47 +187,25 @@ public TableCellRenderer getCustomizedRenderer(DefaultMutableTreeNode o, TableCe
       columns.add(new Column("Time") {
         @Override
         TableCellRenderer createRenderer() {
-          return new EntryCellRenderer() {
-            @Override
-            void render(FlutterLogEntry entry) {
-              appendStyled(entry, TIMESTAMP_FORMAT.format(entry.getTimestamp()));
-            }
-          };
+          return new TimeCellRenderer();
```

Comment:
Hmn, It looks verbose.
If we custom `XXXCellRender` => I think we could do it more simply by passing `cellRender` as a parameter in constructor.
```java
    class Column extends ColumnInfo<DefaultMutableTreeNode, FlutterLogEntry> {
      boolean show = true;
      @NotNull
      private final TableCellRenderer renderer;

      Column(String name, @NotNull TableCellRenderer renderer) {
        super(name);
        this.renderer = renderer;
      }
```

So the code add columns will become:

```java
      columns.add(new Column("Time", new TimeCellRenderer()));
      columns.add(new Column("Sequence", new SequenceCellRenderer()));
      columns.add(new Column("Level", new LevelCellRenderer()));
      columns.add(new Column("Category", new CategoryCellRenderer()));
      columns.add(new Column("Message", new MessageCellRenderer(app.getModule()));
```

OR
```java
      columns.addAll(Arrays.asList(
        new Column("Time", new TimeCellRenderer()),
        new Column("Sequence", new SequenceCellRenderer()),
        new Column("Level", new LevelCellRenderer()),
        new Column("Category", new CategoryCellRenderer()),
        new Column("Message", new MessageCellRenderer(app.getModule()))
      ));
```

Reply:
Nice!  Agreed.